git/git.go

```diff
@@ -0,0 +1,108 @@
+package git
+
+import (
+	"bufio"
+	"canaveral/lib"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+)
+
+// InitRepo initializes a git repo in the current directory
+// ? untested
```

Comment:
Quick note: make the untested indicator red by using `// ! untested` instead of `// ? untested`